app/src/main/java/com/kickstarter/libs/utils/AnimationUtils.kt

```diff
@@ -0,0 +1,23 @@
+package com.kickstarter.libs.utils
+
+import android.view.animation.AlphaAnimation
+import android.view.animation.Animation
+
+object AnimationUtils {
+
+  @JvmOverloads
+  fun disappearAnimation(duration: Long = 300L): Animation {
```

Comment:
so siiiiick!

for posterity: this `@JvmOverloads` makes kotlin export two functions, one with the default and one without.